pygmt/io.py

```diff
@@ -0,0 +1,36 @@
+"""
+PyGMT input/output (I/O) utilities.
+"""
+import xarray as xr
+
+
+def process_output_grid(grid_name, tmpfile_name):
```

Comment:
Basing this on [`xarray.load_dataarray`](https://xarray.pydata.org/en/v0.19.0/generated/xarray.load_dataarray.html) at https://github.com/pydata/xarray/blob/a21dead8621480b608827355b50e85af6f88f185/xarray/backends/api.py#L247

```suggestion
def load_datagrid(filename_or_obj, **kwargs):
```